CHANGELOG.md

```diff
@@ -14,58 +13,62 @@
 ## [0.7.1]
 
 ### Added
+- Added skip-duplicate-runs #423
+- Bump: terraform 0.13.4 ~> 0.13.5 #445
+- Bump: consul 1.8.4 ~> 1.8.5 #445
+- Bump: vault 1.5.4 ~> 1.6.0 #445
+- Bump: nomad 0.12.5 ~> 0.12.8 #445
+- Bump: packer 1.6.4 ~> 1.6.5 #445
 
+### Changed
+- Using random token instead of 'master' for Hashicorp products #394
+
+## [0.7.1]
```

Comment:
`0.7.1` presented twice in the changelog

Reply:
It looks there that several changelog changes added to the wrong section

Reply:
Fixed the bug

Reply:
Was due to incoming changes. Fixed now!